tests/pr_test.go

```diff
@@ -85,7 +85,24 @@ func TestRunBankingSolutions(t *testing.T) {
 			},
 		})
 
-		output, err := options.RunTest()
+		// Ignore updates - expected behaviour - as trying to update the ci/cd pipeline with the assistant ID in the subsequent applies.
```

Comment:
Is there a GIT issue tracking this limitation? I assume this is going to be fixed and is tracked somewhere?

Reply:
I am not aware of the GIT issue... we haven't wrote the core code of this module...

Reply:
Which resources are updated on subsequent apply? Is it ibm_cd_tekton_pipeline_property?